bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -261,55 +261,55 @@ def __init__(self, config, yolo):
         :param yolo: An YoloHandler implementation that runs the yolo network
         """
         # Set the yolo network
-        self.yolo = yolo
+        self._yolo = yolo
         # Set the config. Not needed at the moment
-        self.config = config
+        self._config = config
 
     def set_image(self, image):
         """
         Set a image for yolo. This is cached.
         :param image: current vision image
         """
-        self.yolo.set_image(image)
+        self._yolo.set_image(image)
 
     def get_candidates(self):
         """
         :return: all found ball candidates
         """
-        return self.yolo.get_candidates()[0]
+        return self._yolo.get_candidates()[0]
 
     def compute(self):
         """
         Runs the yolo network
         """
-        self.yolo.predict()
+        self._yolo.predict()
 
 class YoloGoalpostDetector(CandidateFinder):
     """
     A goalpost detector using the yolo neural network
     """
-    def __init__(self, yolo):
+    def __init__(self, yolo):  # TODO Jan: Why does this class have no config param? See YoloBallDetector...
```

Comment:
As well as here...

Reply:
The config param is currently unsued, but I added it for compatibility reasons.